packages/framework/src/Form/Admin/PriceList/PriceListFormType.php

```diff
@@ -113,7 +113,7 @@ public function configureOptions(OptionsResolver $resolver): void
     public function checkDateValidity(PriceListData $priceListData, ExecutionContextInterface $context): void
     {
         if ($priceListData->validTo < $priceListData->validFrom) {
-            $context->buildViolation('"Valid to" must be greater than "Valid from"')
+            $context->buildViolation(t('"Valid to" must be greater than "Valid from"', [], 'validators'))
```

Comment:
nitpick, `Translator::VALIDATOR_TRANSLATION_DOMAIN` should be used. I am wondering whether there is a phpstan rule that would add at least a warning when we use a hardcoded string that is already defined in a constant in the codebase :thinking:

Reply:
https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZHNuNXphdG1vdTVhOHc5N2l3bHQ1cWVtNXF2azkzd3FzczF2bHFjayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/etSwiUkXNhTB6/giphy.gif

Reply:
Well, I'm not sure how the rule should behave. What if we have `SliderItemFormType::SCENARIO_CREATE = 'create';` and I create new button 
```
->add('create', SubmitType::class);
```

How should it detect whether it makes sense to warn about this value or not?

Maybe we could enforce the use of the constant in the `t` function... but it does not bother me that much 🙂 

Anyway, I have changed it